webui/react/src/utils/time.ts

```diff
@@ -0,0 +1,29 @@
+// @ts-ignore
+import humanizeDuration from 'humanize-duration';
+
+import { Experiment } from 'types';
+
+// Experiment duration (naive) in miliseconds
+export const experimentDuration = (experiment: Experiment): number => {
+  const endTime = experiment.endTime ? new Date(experiment.endTime) : new Date();
```

Comment:
suggestion: Since `endTime` can be `undefined` or a `string`, should just be able to do
```
const endTime = new Date(experiment.endTime);
```

Reply:
looks like we can't. Date() doesn't accept 'undefined' as a parameter (or rather non-parameter)